src/components/QuickOpenModal.js

```diff
@@ -96,12 +96,17 @@ export class QuickOpenModal extends Component<Props, State> {
   }
 
   componentDidUpdate(prevProps: Props) {
+    const nowEnabled = !prevProps.enabled && this.props.enabled;
+    const queryChanged = prevProps.query !== this.props.query;
```

Comment:
Not really needed to strict compare a value unless you expect it to be 0 or 1, this one is going to be a string.

Reply:
It is a string. :) I dunno, I have the habit of always using strict comparisons since they're more precise. Are there any performance decrements?

Reply:
Strict comparison should be faster

Reply:
Personally I prefer the strict checks for all things except for when you're comparing to `null`/`undefined` where `if (foo == null)` is equivalent to `if (foo === null || foo === undefined)` but yeah. If I had a thing I'd prefer here it'd be this but only because I like explicitly plucking the props/prevProps I care about.

```js
componentDidUpdate({ enabled: prevEnabled, query: prevQuery }) {
  const { enabled, query } = this.props;
  const nowEnabled = !prevEnabled && enabled;
  const queryChanged = prevQuery !== query;
  // ...
}